xgcm/test/test_grid.py

```diff
@@ -784,20 +784,62 @@ def test_boundary_kwarg_same_as_grid_constructor_kwarg():
 
 @pytest.mark.parametrize(
     "metric_axes,metric_name",
-    [("X", "dx_t"), ("Y", "dy_ne"), (["Y", "X"], "dy_n")],
+    [(["Y", "X"], "area_n")],
```

Comment:
Hi, so I just wanted to get one set of metric axes and metric name going with all the parameters before I put in other inputs because testing takes so long for me X_____x So basically I will be trying out one pair of metric axes and name at a time with all of the parameters, and then for the final PR, I'll include all metric axes and name pairs.